lib/proc/proc_tokenize.c

```diff
@@ -54,6 +54,35 @@ fin_tokens(grn_ctx *ctx, grn_obj *tokens)
   GRN_OBJ_FIN(ctx, tokens);
 }
 
+static void
+output_only_tokens(grn_ctx *ctx, grn_obj *tokens, grn_obj *lexicon)
+{
+  size_t i, n_tokens, n_elements;
+  n_tokens = GRN_BULK_VSIZE(tokens) / sizeof(tokenize_token);
+  n_elements = 1;
```

Comment:
```suggestion
  size_t i, n_tokens;
  n_tokens = GRN_BULK_VSIZE(tokens) / sizeof(tokenize_token);
```

This variable is not used.